Source/WebKit/UIProcess/API/wpe/WebKitWebViewWPE.cpp

```diff
@@ -50,19 +50,144 @@ gboolean webkitWebViewRunFileChooser(WebKitWebView*, WebKitFileChooserRequest*)
     return FALSE;
 }
 
-void webkitWebViewMaximizeWindow(WebKitWebView*, CompletionHandler<void()>&& completionHandler)
+struct WindowStateEvent {
```

Comment:
```suggestion
struct ToplevelStateEvent {
```

Reply:
Let's use Toplevel instead of Window everywhere, since it better matches the WPE API name